data/categories.txt

```diff
@@ -1553,7 +1557,7 @@ cs:Autobusové nádraží|zastávka|bus|doprava
 sk:Autobusové nástupište|zastávka|bus|doprava
 nl:Busstation|2bus|station|4transport
 zh-Hant:1巴士|巴士站|交通|運輸
-pl:2Stacja autobusowa|Zajezdnia|transport
+pl:Dworzec autobusowy|autobus|transport
```

Comment:
Unfortunately in Polish there is no such phrase  as "stacja autobusowa". Synonym for station is called "dworzec" - "dworzec autobusowy" for bus station and "dworzec kolejowy" for railway station. "Stacja kolejowa" (or "stacja metra" for metro) is the name of place where trains stops, but bus stop is called "przystanek autobusowy". Previous translator translate this phrase too literally.
Should I ignore this and add "stacja" as synonym anyway?